src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/service/FeatureTogglerService.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.service;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class FeatureTogglerService {
+
+    private FeatureToggler featureToggler;
+
+    public FeatureTogglerService(FeatureToggler featureToggler) {
+        this.featureToggler = featureToggler;
+    }
+
+    public boolean getValueForMakeAnApplicationFeature() {
+        return featureToggler.getValue("make-an-application-feature", false);
```

Comment:
We do not need separate service to wrap only one line of code, please remove and use inline `featureToggler.getValue("make-an-application-feature", false)` in handlers

Reply:
Now the FeatureToggler call is made inline in the handler.